splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -704,17 +704,22 @@ def generate(
         output_directory, ta_name, "default", app_conf_lib.APP_CONF_FILE_NAME
     )
     app_conf.read(output_app_conf_path)
-    should_be_visible = True if global_config else False
-    if global_config and global_config.meta.get("checkForUpdates") is False:
-        check_for_updates = False
-    else:
-        check_for_updates = True
+    should_be_visible = False
+    check_for_updates = "true"
+    supported_themes = ""
+    if global_config:
+        should_be_visible = True
+        if global_config.meta.get("checkForUpdates") is False:
+            check_for_updates = "false"
+        if global_config.meta.get("supportedThemes") is not None:
+            supported_themes = ", ".join(global_config.meta["supportedThemes"])
```

Comment:
I tried the following:

```
poetry run ucc-gen init --addon-name test_addon --addon-rest-root test_addon --addon-display-name TestAddonForSplunk --addon-input-name test_input_name --overwrite
poetry run ucc-gen build --source test_addon/package
poetry run ucc-gen package --path output/test_addon
```

and it does not work when I switch to the dark theme.

I suspect that's because extra space that appears in `app.conf` file.

Reply:
It does not work with docker run as well

Reply:
I downloaded the build generated by the pipeline and installed it on Splunk and it works.

Reply:
I think the `ucc-gen init` doesn't take into account the `scripts/build_ui.sh` file to build the UI files.

Reply:
`scripts/build_ui.sh` might have contained previously built JS files and cleaning the directory helped me.